h/static/styles/login-form.scss

```diff
@@ -0,0 +1,148 @@
+.login-form {
+  font-size: $body2-font-size;
+
+  margin-top: 57px; // When you add the height and margin-top of
+                    // .logged-out-navbar to this, it makes 100px between the
+                    // top of the document and the top of the login form.
+
+  max-width: 500px;
+  margin-left: auto;
+  margin-right: auto;
+
+  // Keep a little white space around the form, even on small screens.
+  // This is the same size as the margin-left on the logged-out-navbar.
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.login-form__heading {
+  font-size: $font-size-heading2;
+  font-weight: normal;
```

Comment:
This should probably be moved to a new `$variable`, it seems that `normal` weight headings are part of our style now